### PR TITLE
mnemosyne: add googletrans and gtts python dependencies

### PR DESCRIPTION
**Pull request by unode**
###### Motivation for this change
Fixes `mnemosyne`'s crash at launch due to missing `googletrans` #80430  and `gtts` #81149 .
The two linked issues should be addressed and merged before this one.

The runtime error of `mnemosyne` is also mentioned in #80924 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by unode** [comment]
pinging @jonringer as reviewer of `mnemosyne`'s runtime issue in #80924

File: pkgs/games/mnemosyne/default.nix [hunk 1]
```diff
@@ -17,6 +17,8 @@ python.pkgs.buildPythonApplication rec {
   buildInputs = [ anki ];
 
   propagatedBuildInputs = with python.pkgs; [
+    googletrans
+    gtts
```
Comment (marsam) [inline pkgs/games/mnemosyne/default.nix]: you need to package [`gtts`](https://pypi.org/project/gTTS/)

**Review by marsam — CHANGES_REQUESTED** [review]

**Comment by jonringer** [comment]
@GrahamcOfBorg eval

**Review by jonringer — APPROVED** [review]
diff LGTM
does fix executable launch

```
[2 built, 6 copied (9.8 MiB), 3.2 MiB DL]
https://github.com/NixOS/nixpkgs/pull/81150
1 package built:
mnemosyne
```

**Comment by jonringer** [comment]
@GrahamcOfBorg build mnemosyne

**Comment by unode** [comment]
Not sure what to do about the Darwin failures.
Anything holding the merge?

**Comment by jonringer** [comment]
failure is from a different package, I think this is fine.

**Pull request merged**